### ISSUE TITLE
Installation Directory on Windows

### ISSUE THREAD

**Issue by jbozeman** [calebegg/proof-pad-classic#51]
The 64-bit version automatically selects C:\Program Files(x86) as the install location. It should choose C:\Program Files\ instead.

**Comment by calebegg** [comment]
I agree. I'm not sure what the best way to do this is with NSIS. Their variable for the 64-bit folder is `$PROGRAMFILES64` I'd like to avoid having two versions of the script. Would you be interested in looking into this? I'm kind of stumbling around in the dark with NSIS (if the .nsi source code wasn't evidence enough).

It might also be nice if the NSIS script could create both 32 and 64 bit installers in one go (especially if it would make solving this problem easier). Currently, I have an acl286/ and an al264/ folder, and I just rename one of them to acl2/ when I generate that one's installer, then rename the installer and change it back. Kind of a pain.

Anyway, if you can't get to it, I'll get to it eventually.
